src/components/navigation/TopNavMenu.tsx

```diff
@@ -45,8 +45,8 @@ export const TopNavMenu = () => {
         onClick={handleTouch}
         sx={styles.button}
       >
+        {user && <Box component={'img'} src={userImage} sx={styles.profileImage} />}
```

Comment:
it's better to use `userImage &&` here in case if user exists but doesn't have an image

Reply:
users have default images in case they don't select one. But its good point I will change it to `userImage`